client/src/LoginPage.js

```diff
@@ -1,22 +1,60 @@
 import React, { Component } from "react";
+import { Alert } from 'reactstrap';
 
 export default class Login extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {email: "",
+                      password: "",
+                      renderAlert: false};
+    }
+
+    handleEmailFieldChange = (e) => {
+        this.setState({email: e.target.value});
+    }
+
+    handlePasswordFieldChange = (e) => {
+        this.setState({password: e.target.value});
+    }
+
+    handleLogin = (e) => {
+        e.preventDefault(); // Prevents the form from submitting when Submit button is clicked
+
+        const email =  this.state.email.trim();
+        const password =  this.state.password.trim();
+        if (email.localeCompare("") === 0 || password.localeCompare("") === 0) {
+            this.setState({renderAlert: true});
+        } else {
+            this.setState({renderAlert: false});
+            // Call backend API with user input
+        }
+    }
+
     render() {
+        alert = (
```

Comment:
I would define this outside the render function, or just use it directly in line 
this.state.renderAlert && <Alert> . ... </Alert>

Reply:
I will use it directly in-line. 
But if I had wanted to define it outside the render function, 'alert' would not have been recognized as a variable inside of the render function when I want to reference it. Would I have to add it to the state in the constructor? Or is there another way to do that?

Reply:
a function that returns the jsx component.